openapi-cli/src/main/java/io/ballerina/openapi/generators/client/BallerinaAuthConfigGenerator.java

```diff
@@ -186,6 +189,15 @@ public boolean isHttpOROAuth() {
         return httpOROAuth;
     }
 
+    /**
+     * Returns tokenUrl value in OAuth clientCredentials.
+     *
+     * @return {@link String}    TokenUrl
+     */
+    public String getClientCredGrantTokenUrl() {
```

Comment:
We can also send a boolean here since this function is only used to know whether the token URL is given. WDYT ?

Reply:
Done